### PR TITLE
Integrated Ripple animation button

### PR DESCRIPTION
**Pull request by BhavaniMM**
Description:

Integrated Ripple animation button using using Lottie-react-native.

Testing:

IOS:

https://github.com/Ganesh-softsuave/Affitu/assets/135315139/4d751f6f-3907-4186-998d-8a65efed4f4a

Android:

https://github.com/Ganesh-softsuave/Affitu/assets/135315139/531af2d7-5128-478c-8f04-9cacb8c9ce6e

### CODE REVIEW

**Review by dilipkrishna97 — DISMISSED** [review]

File: ios/GoogleService-Info.plist [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```
Comment (Ganesh-softsuave) [inline ios/GoogleService-Info.plist:3]: Please revert this file. This file has Keys and client Id. we should not commit this

File: src/components/App/index.tsx [hunk 2]
```diff
@@ -9,10 +9,12 @@ import React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
 import {API_URL, ENV} from 'react-native-config';
 import {useTranslation} from 'react-i18next';
+
 import {colorType} from 'lib/theme/colors';
 import {ThemeContext} from '@src/context/themeContext';
 import ViewModel from './ViewModel';
 import {ViewModelReturnType} from './types';
+import RippleAnimationButton from '../RippleAnimationButton';
```
Comment (Ganesh-softsuave) [inline src/components/App/index.tsx:17]: Please use absolute path

File: src/components/RippleAnimationButton/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  const styles = rippleAnimationStyle(themeColor.color);
+
+  return (
+    <View style={styles.rippleBtnContainer}>
+      <LottieView
+        source={props.filePath}
+        autoPlay
+        loop
+        style={styles.lottieViewStyle}
+        {...testID('ripple_lottie_view')}
```
Comment (Ganesh-softsuave) [inline src/components/RippleAnimationButton/index.tsx:30]: Please add animation name as well in testId

File: lib/theme/colors/index.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import lightTheme from './light';
 
-export {default as dark} from './light';
-export {default as light} from './dark';
+export {default as light} from './light';
```
[the remark this answers is not in the document]
Comment (BhavaniMM) [reply lib/theme/colors/index.ts:3]: Before default name and file names were different. Added correct default name as per file name.

**Review by Ganesh-softsuave — DISMISSED** [review]

**Review by Ganesh-softsuave — APPROVED** [review]

**Pull request merged**